ts/private/ts_project_options_validator.js

```diff
@@ -131,6 +131,17 @@ function main(_a) {
             )
         }
     }
+    function check_nocheck() {
+        if (attrs.isolated_typecheck) {
+            var optionVal = getTsOption('isolatedDeclarations')
+            if (!optionVal) {
+                failures.push(
+                    'attribute isolated_typecheck=True requires compilerOptions.isolatedDeclarations=true'
```

Comment:
also mention here that setting `compilerOptions.isolatedDeclarations=true` to True can cause compile errors because of <insert reasons> and that it is recommended to front-load that change separately from flipping `isolated_typecheck` to True